src/openzeppelin/tests/token/test_erc20.cairo

```diff
@@ -260,12 +305,18 @@ fn test_transfer_from() {
 #[available_gas(2000000)]
 fn test_transfer_from_doesnt_consume_infinite_allowance() {
     let mut state = setup();
-    set_caller_address(OWNER());
+    testing::set_caller_address(OWNER());
     ERC20Impl::approve(ref state, SPENDER(), BoundedInt::max());
+    testing::pop_log_raw(ZERO());
 
-    set_caller_address(SPENDER());
+    testing::set_caller_address(SPENDER());
     ERC20Impl::transfer_from(ref state, OWNER(), RECIPIENT(), VALUE);
 
+    let event = testing::pop_log::<Transfer>(ZERO()).unwrap();
+    assert(event.from == OWNER(), 'Invalid from');
+    assert(event.to == RECIPIENT(), 'Invalid to');
+    assert(event.value == VALUE, 'Invalid value');
+
```

Comment:
We're already testing events from `transfer_from` in `test_transfer_from`. Do we need this here?

Reply:
Good call.